tests/RuleSet/LenientParsingTest.php

```diff
@@ -11,16 +11,7 @@
 use Sabberworm\CSS\Settings;
 
 /**
- * @covers \Sabberworm\CSS\CSSList\Document
- * @covers \Sabberworm\CSS\Parser
- * @covers \Sabberworm\CSS\RuleSet\DeclarationBlock
- * @covers \Sabberworm\CSS\Rule\Rule
- * @covers \Sabberworm\CSS\Value\CSSString
- * @covers \Sabberworm\CSS\Value\CalcFunction
- * @covers \Sabberworm\CSS\Value\Color
- * @covers \Sabberworm\CSS\Value\LineName
- * @covers \Sabberworm\CSS\Value\Size
- * @covers \Sabberworm\CSS\Value\URL
+ * @coversNothing
```

Comment:
Agree it's generally not clear which of the classes this covers - but doesn't it at least cover `Settings` to some extent?

Reply:
It _calls_ code in `Settings` (and uses it). It does not test whether `Settings` actually work correctly, though, and the point of the tests is not testing `Settings` (as far as I understand them), but how other code works depending on the settings.